src/pages/Tags/classes.module.scss

```diff
@@ -0,0 +1,101 @@
+@import 'styles/variables';
+@import 'styles/mixins';
+
+.tagsHeader {
+  display: grid;
+  grid-template-columns: max-content auto max-content max-content;
```

Comment:
[blocker]
 You only have two things in the header container, but you've split the content into four parts

`grid-template-columns: max-content max-content;`